src/app/tests/suites/DL_UsersAndCredentials.yaml

```diff
@@ -2051,6 +2051,22 @@ tests:
               - name: "NextUserIndex"
                 value: 2
 
+    - label: "Make sure a LockUserChange event was generated"
+      command: "readEvent"
+      event: "LockUserChange"
+      # I wish there were a way to not hardcode this 25, but it's experimentally
+      # determined: doing a read without an eventNumber filter here shows 24
+      # LockUserChange events before this removal.
```

Comment:
Could we add a TODO/issue here? Maybe we could have some `reset event count` somewhere to make this a lot more predictable. 24/25 feels very much arbitrary.

Reply:
I can file an issue, but fundamentally the issue is that the Door Lock app does not implement any non-standard clusters that would let us do that..

I filed https://github.com/project-chip/connectedhomeip/issues/33806